app/assets/javascripts/surveys.js

```diff
@@ -795,7 +847,20 @@
     currentBreadcrumb: function () { return $('.govuk-breadcrumbs').text() || '' },
     currentSection: function () { return $('meta[name="govuk:section"]').attr('content') || '' },
     currentThemes: function () { return $('meta[name="govuk:themes"]').attr('content') || '' },
-    currentOrganisation: function () { return $('meta[name="govuk:analytics:organisations"]').attr('content') || '' }
+    currentOrganisation: function () { return $('meta[name="govuk:analytics:organisations"]').attr('content') || '' },
+    currentTlsVersion: function () {
```

Comment:
As there's a decent amount of logic going on in this method I think we should have some tests around it.